webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/eventlog/EventLogFactory.scala

```diff
@@ -893,6 +905,35 @@ class EventLogFactoryImpl(
     ModifyGlobalProperty(eventLogType,eventLogDetails)
   }
 
+ override def getPromoteToRelayFromDiff(
+      id             : Option[Int] = None
+    , modificationId : Option[ModificationId] = None
+    , principal      : EventActor
+    , promotedNode        : Node
+    , creationDate   : DateTime = DateTime.now()
+    , severity       : Int = 100
+    , reason         : Option[String]
+ ) : PromoteNode = {
+
+   val details = EventLog.withContent{
+     scala.xml.Utility.trim(
+       <node changeType="modify" fileFormat={Constants.XML_CURRENT_FILE_FORMAT.toString}>
+         <id>{promotedNode.id.value}</id>
+         <name>{promotedNode.}</name>
+         <state>{promotedNode.state}</state>
+       </node>
```

Comment:
I'm not sure about what we want to display in this eventlog

Reply:
I think just the nodeid/hostname is enought